aws/resource_aws_lb_test.go

```diff
@@ -1956,6 +1987,55 @@ resource "aws_lb" "test" {
 `, rName))
 }
 
+func testAccAWSLBConfig_IPv6(rName string) string {
+	return composeConfig(
+		testAccAvailableAZsNoOptInConfig(),
+		fmt.Sprintf(`
+resource "aws_vpc" "test" {
+  assign_generated_ipv6_cidr_block = true
+  cidr_block                       = "10.10.10.0/25"
+
+  tags = {
+    Name = "tf-acc-test-load-balancer"
+  }
+}
+
+resource "aws_internet_gateway" "gw" {
+  vpc_id = aws_vpc.test.id
+
+  tags = {
+    Name = "main"
+  }
+}
+
+resource "aws_subnet" "test" {
+  availability_zone = data.aws_availability_zones.available.names[0]
+  cidr_block        = cidrsubnet(aws_vpc.test.cidr_block, 2, 0)
+  ipv6_cidr_block   = cidrsubnet(aws_vpc.test.ipv6_cidr_block, 8, 16)
+  vpc_id            = aws_vpc.test.id
+
+  tags = {
+    Name = "tf-acc-test-load-balancer"
+  }
+}
+
+resource "aws_lb" "test" {
+  name                       = %[1]q
+  load_balancer_type         = "network"
+  enable_deletion_protection = false
+
+  subnet_mapping {
+    subnet_id    = aws_subnet.test.id
+    ipv6_address = cidrhost(cidrsubnet(aws_vpc.test.ipv6_cidr_block, 8, 16), 5)
+  }
+
+  tags = {
+    Name = "TestAccAWSALB_ipv6address"
+  }
+}
```

Comment:
locally this test works great, but in TeamCity seems the gateway creation takes a bit longer as I'm seeing this error:
```
resource_aws_lb_test.go:202: Step 1/2 error: Error running apply: 
        Error: error creating network Load Balancer: InvalidSubnet: VPC vpc-009bd6f8aa5641496 has no internet gateway
        	status code: 400, request id: 6dbc33f4-ca80-499e-a4f9-b95d040e5e44
```
so adding the dependency here should safeguard it 👍 
```suggestion
  depends_on = [aws_internet_gateway.gw]
}
```